icons-gallery/src/components/Preview.tsx

```diff
@@ -0,0 +1,13 @@
+import { Devvit } from '@devvit/public-api';
+import CustomPostComponent = Devvit.CustomPostComponent;
+
+export const Preview: CustomPostComponent = (): JSX.Element => {
+  return (
+    <blocks height={'tall'}>
```

Comment:
:no_bell: I don't think we need the braces on these plain strings and probably don't want users thinking they do.